packages/elements/src/pagination/index.ts

```diff
@@ -637,27 +610,24 @@ export class Pagination extends BasicElement {
    * @returns {void}
    */
   private onKeyDown (event: KeyboardEvent): void {
+
     if (event.defaultPrevented) {
       return;
     }
 
-    const newInputValue = Number(this.input.value);
-
+    // Handle keyboard shortcuts
     switch (event.key) {
       case 'Enter':
-        this.updatePageInput();
-        this.input.blur();
-        break;
       case 'Tab':
```

Comment:
Why do you need Tab here at all? When you tab away from input, blur event will fire and the page is set anyway?

Reply:
Yes, your are correct. We do not need `Tab` here. Thanks